home-page.html

```diff
@@ -11,11 +11,63 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    <script src="script.js" defer></script>
+    <script src="contributors.js" defer></script>
+    <link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
+<body id="home-page">
     <section class="background-image-wrapper">
         <div class="over-lay">
-            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+                <div id="pop-up-menu-wrapper">
+                    <i class="fa-solid fa-xmark" id="cancel"></i>
+                <div class="pop-up-menu">
+                    <ul class=".pop-up-list">
+                        <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
+                        <li class="pop-up-li"><a class="page-link" href="index.html">About</a></li>
+                        <li class="pop-up-li">Program</li>
+                        <li class="pop-up-li">Join</li>
+                        <li class="pop-up-li">Sponsor</li>
+                    </ul>
+                </div>
+                </div></header>
+            <header class="about-desk-head">
+                <div class="sign-in-bar">
+                    <i class="fa-brands fa-facebook"></i>
+                    <i class="fa-brands fa-twitter"></i>
+                    <ul>
+                        <li><a href="">Logout</a></li>
+                        <li><a href="">Mypage</a></li>
+                        <li><a href="">Extras</a></li>
+                    </ul>
+                </div>
+                <div class="white-part">
+                    <div class="cc-image">
+                        <i class="fa-solid fa-closed-captioning"></i>
+                    </div>
+                    <div class="text">
+                        <p>
+                            CC GLOBAL
+                        </p>
+                        <p>
+                            Summit 2015
+                        </p>
+                        <p class="part-two dates">
+                            14th - 17th oct 2015 | Seoul, Korea
+                        </p>
+                    </div>
+                    <div class="menu-list">
+                        <ul class="menu-list-list">
+                            <li><a href="index.html">About</a></li>
+                            <li><a href="">Program</a></li>
+                            <li><a class="page-link" href="home-page.html">Home</a></li>
```

Comment:
- [ ] Great job so far but I think in your home page for the desktopversion it would be great if you fix spaces here so please decrease the margin/padding between the elements  since your design it should be identical to the original design so please fix the spaces here according to the requirements 🤩🔥♻🙏🏻✨

- [ ] I think it would be great if you decrease the width of the text here I can see it takes the whole page width but I should just take smaller width as you can see in the given design so kindly fix it 

- [ ] I can see there is a space in the side so please make sure you made in the body take the border box sizes to not let spaces in the side shown as you can see in the image below so kindly remove it  ⬇✨🙏🏻🙏🏻
- Your design
![image](https://user-images.githubusercontent.com/58553711/219919166-0d4537a2-a3e0-4dcb-bfd9-f33e1757dcab.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/219919658-df27a8df-a359-4d7f-8cba-40a2d483031c.png)